perceval/backends/core/git.py

```diff
@@ -338,6 +338,10 @@ def _pre_init(self):
 
         if self.parsed_args.git_log:
             git_path = self.parsed_args.git_log
+        elif self.parsed_args.base_path:
```

Comment:
I think the full `if` block can be reduced to the next code. Thus, we don't repeat ourselves. What do you think?

```
if self.parsed_args.git_log:
    git_path = self.parsed_args.git_log
elif self.parsed_args.git_path:
    git_path = self.parsed_args.git_path
else:
    if self.parsed_args.base_path:
        base_path = self.parsed_args.base_path
    else:
        base_path = os.path.expanduser('~/.perceval/repositories/')   

    processed_uri = self.parsed_args.uri.lstrip('/')
    git_path = os.path.join(base_path, processed_uri) + '-git'
```